### PR TITLE
wip: Create PersonalBio component

### PR DESCRIPTION
**Pull request by arnest00**
### Description
- Heading reads "About `firstName`"
- Bio listed underneath heading
- Styles matches design, outside of layout

### Spec
Designs: [Designs](https://www.figma.com/file/kswpPSHH2h5njVyfGDbIHk/FSA-Capstone-Project-2022)
See Story: [197](https://sparkbox.atlassian.net/browse/FSA22V1-197)

### Validation
* [x] This PR has code changes, and our linters still pass.
* [x] This PR affects production code, so it was browser tested
* [x] This PR has new code, so new tests were added or updated, and they pass.
* [x] Changes are browser tested. This includes functionality, accessibility, responsiveness, and design.

#### To Validate

1. Make sure all PR Checks have passed.
2. Pull down `feat--personal-bio` branch, run `npm install` then `npm run dev`...
3. ...or open the deploy preview.
4. Navigate to any team member's page and verify rendered bio. Team members without a bio will not render this component on their page.

### CODE REVIEW

**Comment by arnest00** [comment]
Tagging @QuinnrElder and @howema for your review on this one!

**Review by howema — APPROVED** [review]
Looks great! Merging

File: components/PersonalBio.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import Show from './Show';
+
+export default function PersonalBio({ name, bio }) {
+  return (
+    <Show when={bio}>
+      <section aria-labelledby="personal-bio" className="cmp-personal-bio">
+        <h3 id="personal-bio" className="cmp-personal-bio__header">
+          About
```
Comment (QuinnrElder) [inline components/PersonalBio.jsx:8]: Already implementing things learned this morning!! #KillingIt

File: styles/components/_personal-bio.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  letter-spacing: -0.02em;
+  width: 100%;
+
+  &__header {
+    font-family: font-family("sans-extended");
+    font-size: font-size("lg");
+    font-weight: 600;
+    line-height: calc($header-line-height / 2);
+    margin: 0;
+    margin-bottom: space("md");
```
Comment (QuinnrElder) [inline styles/components/_personal-bio.scss:16]: In mobile view, it looks like the margin spacing might be differing from the component (skills) above.

Screen Shot 2022-05-19 at 10 59 04 AM

Comment (QuinnrElder) [reply styles/components/_personal-bio.scss:16]: Same in Ipad ~

**Review by QuinnrElder — APPROVED** [review]
Talked with Osmond ~ "we’ll be adding mobile view changes to the app after we get some more clarification from Merani. We’ll otherwise get all the other general layout stuff done in a future card."

This clarification above resolves my outstanding questions for this PR.

LGTM!

**Pull request merged**